src/ui/components/topicButton/TopicButton.tsx

```diff
@@ -17,7 +17,7 @@ export const VuiTopicButton = ({ children, className, href, onClick, title, ...r
     <>
       {title && (
         <>
-          <VuiTitle size="s">
```

Comment:
Bring this back to "s" though? (this was changed by me originally to reduce the size)